### PR TITLE
merge develop & resolve conflict

### PR DESCRIPTION
**Pull request by phucvarmeta**

### CODE REVIEW

File: packages/web/src/components/unstake/unstake-liquidity/UnstakeLiquidity.tsx [hunk 1]
```diff
@@ -44,25 +48,33 @@ const UnstakeLiquidity: React.FC<UnstakeLiquidityProps> = ({
...
+            ? t("UnstakePosition:btn.selectPosi")
+            : t("UnstakePosition:btn.unstake")
+        }
         style={{
           hierarchy: ButtonHierarchy.Primary,
           fullWidth: true,
         }}
         className="button-confirm"
-        disabled={checkedList.length === 0}
+        // disabled={checkedList.length === 0}
```
Comment (AkydA) [inline packages/web/src/components/unstake/unstake-liquidity/UnstakeLiquidity.tsx:77]: is condition changed?

File: packages/web/src/components/unstake/unstake-position-modal/UnstakePositionModal.tsx [hunk 2]
```diff
@@ -188,24 +189,35 @@ const UnstakePositionModal: React.FC<Props> = ({
...
+                <Trans
+                  ns="UnstakePosition"
+                  i18nKey="confStakeModal.warning.content"
+                  values={{
+                    currentPercent,
+                    swapFeePercent,
+                  }}
+                >
+                  Your APR will reduce from {currentPercent} →{" "}
+                  <span className="unstake-percent">{swapFeePercent}</span>
```
Comment (AkydA) [inline packages/web/src/components/unstake/unstake-position-modal/UnstakePositionModal.tsx:213]: it maybe redundant string

File: packages/web/src/components/unstake/unstake-liquidity/UnstakeLiquidity.tsx (hunk 1 above)
Comment (phucvarmeta) [reply packages/web/src/components/unstake/unstake-liquidity/UnstakeLiquidity.tsx:77]: Nope i will uncomment it

File: packages/web/src/components/unstake/unstake-position-modal/UnstakePositionModal.tsx (hunk 2 above)
Comment (phucvarmeta) [reply packages/web/src/components/unstake/unstake-position-modal/UnstakePositionModal.tsx:213]: This is not redundant, Trans component treats it like a string template

**Pull request merged**